cmd/nerdctl/builder_build_test.go

```diff
@@ -498,3 +498,50 @@ func TestBuildNetworkShellCompletion(t *testing.T) {
 	networkName := "default"
 	base.Cmd(gsc, "build", "--network", "").AssertOutContains(networkName)
 }
+
+func TestBuildAttestation(t *testing.T) {
+	testutil.DockerIncompatible(t) // ref:https://github.com/docker/buildx/issues/1655
```

Comment:
Can we just create `docker-container` builder in GHA?

Reply:
Thanks @AkihiroSuda 

The test is docker-compatible by a docker-builder created in a test case. :-)